File: Assets/AltUnityTester/AltUnityDriver/Notifications/INotificationCallbacks.cs

namespace Altom.AltUnityDriver.Notifications
{
    public interface INotificationCallbacks
    {
        void SceneLoadedCallback(AltUnityLoadSceneNotificationResultParams altUnityLoadSceneNotificationResultParams);
        void HierarchyChangesCallback(AltUnityHierarchyNotificationResultParams altUnityHierarchyNotificationResultParams);
        void SceneUnloadedCallback(string sceneName);
    }
}

File: Bindings~/java/src/main/java/ro/altom/altunitytester/Notifications/INotificationCallbacks.java
package ro.altom.altunitytester.Notifications;

public interface INotificationCallbacks {
    void SceneLoadedCallBack(AltUnityLoadSceneNotificationResultParams altUnityLoadSceneNotificationResultParams);
    void HierarchyChangedCallBack(AltUnityHierarchyChangedNotificationResultParams altUnityHierarchyChangedNotificationResultParams);
    void SceneUnloadedCallBack(String sceneName);
}